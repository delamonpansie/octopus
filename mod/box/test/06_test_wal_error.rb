#!/usr/bin/ruby

$: << File.dirname($0) + '/lib'
require 'run_env'

class Env < RunEnv
  def config
    super + <<EOD
wal_dir = "/"
EOD
  end
end

Env.connect_eval do
  ping
  log_try { insert [1, 2, "abc", "def"] }
  log_try { insert [1, 2, "abc", "def"] }
end
