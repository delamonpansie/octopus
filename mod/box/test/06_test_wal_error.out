# box.ping()
:pong

# box.insert([1, 2, "abc", "def"])
Failed with: {code: 0x2702, message: 'unable write wal row'}
# box.insert([1, 2, "abc", "def"])
Failed with: {code: 0x2702, message: 'unable write wal row'}
